### PR TITLE
[406089][ca] no ca for visibility keywords for method declaration

### PR DESCRIPTION
**Pull request by cdietrich**
[406089][ca] no ca for visibility keywords for method declaration

Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=406089

**Comment by cdietrich** [comment]
please review

**Comment by szarnekow** [comment]
I assume we suffer from the same problem for field declarations after 'var' or 'val'?

**Comment by cdietrich** [comment]
Did not check that. Could be a recovery problem as well

**Comment by cdietrich** [comment]
@szarnekow it does not work for fields cause
`org.eclipse.xtend.ide.contentassist.antlr.FlexProposalConflictHelper.existsConflict(String, String, ContentAssistContext)`

says "conflict"

File: org.eclipse.xtend.ide/src/org/eclipse/xtend/ide/contentassist/XtendProposalProvider.java [hunk 1]
```diff
@@ -620,6 +623,25 @@ public void completeXSwitchExpression_Default(EObject model, Assignment assignme
 	}
 	
 	@Override
+	public void complete_CommonModifier(EObject model, RuleCall ruleCall, ContentAssistContext context,
+			ICompletionProposalAcceptor acceptor) {
+		super.complete_CommonModifier(model, ruleCall, context, acceptor);
+		if (model instanceof XtendFunction) {
```
Comment (kthoms) [inline org.eclipse.xtend.ide/src/org/eclipse/xtend/ide/contentassist/XtendProposalProvider.java]: Could you reduce the number of indentation levels, e.g. by quick-return if no XtendFunction and usage of lambdas?

**Pull request closed without merging**